src/components/Live/LiveProvider.js

```diff
@@ -8,7 +8,8 @@ export default class LiveProvider extends Component {
   static defaultProps = {
     code: '',
     noInline: false,
-    language: 'jsx'
+    language: 'jsx',
+    readOnly: false
```

Comment:
I still think we should name this prop `disabled` to match the expectation of the underlying textarea